tests/integration/Router/Endpoints/TrackTest.php

```diff
@@ -131,6 +133,7 @@ public function testItFailsWhenSubscriberIsNotOnProcessedList() {
     $this->entityManager->persist($subscriber);
     $this->entityManager->flush();
     $data->subscriber->setId($subscriber->getId());
```

Comment:
Isn't this line unnecessary now? This line sets subscriber ID, and the line below overwrites the whole subscriber.